crates/store/re_grpc_server/src/lib.rs

```diff
@@ -247,28 +255,33 @@ pub fn spawn_with_recv(
     addr: SocketAddr,
     memory_limit: MemoryLimit,
     shutdown: shutdown::Shutdown,
-) -> re_smart_channel::Receiver<re_log_types::LogMsg> {
-    let (channel_tx, channel_rx) = re_smart_channel::smart_channel(
+) -> (
+    re_smart_channel::Receiver<re_log_types::LogMsg>,
+    crossbeam::channel::Receiver<re_log_types::TableMsg>,
+) {
+    let (channel_log_tx, channel_log_rx) = re_smart_channel::smart_channel(
         re_smart_channel::SmartMessageSource::MessageProxy {
             url: format!("rerun+http://{addr}/proxy"),
         },
         re_smart_channel::SmartChannelSource::MessageProxy {
             url: format!("rerun+http://{addr}/proxy"),
         },
     );
-    let (message_proxy, mut broadcast_rx) = MessageProxy::new_with_recv(memory_limit);
+    let (channel_table_tx, channel_table_rx) = crossbeam::channel::unbounded();
```

Comment:
Why are we using two channels here and elsewhere, instead of just one?

Reply:
Documented in `LogMsg` and `TableMsg`.